devito/propagator.py

```diff
@@ -342,11 +342,16 @@ def sympy_to_cgen(self, stencils):
             stencil = self.convert_equality_to_cgen(equality)
             stmts.append(stencil)
 
+        nest_decl = lambda dec, stmt: cgen.Assign(dec.__str__()[:-1], stmt.rvalue)
+
+        for idx, dec in enumerate(decl):
+            stmts[idx] = nest_decl(dec, stmts[idx])
```

Comment:
I ran all the tests case locally. It all passed. Any idea on why did the travis fail?